src/lwc/rd2ElevateInformation/rd2ElevateInformation.js

```diff
@@ -170,6 +172,7 @@ export default class rd2ElevateInformation extends NavigationMixin(LightningElem
                 this.isElevateCustomer = response.isElevateCustomer;
                 this.permissions.alert = response.alert;
                 this.commitmentURLPrefix = response.commitmentURLPrefix;
+                this.closedStatus = response.closedStatus;
```

Comment:
This function we're assigning `closedStatus` in here has changed a bit over time, and some of the information we're getting back here isn't permission-related. Maybe it's time for this function to be called `getAppConfigurationData` instead?